versioned_docs/version-22.04/developer/developer-gorgone-rebound-mode.md

```diff
@@ -0,0 +1,135 @@
+---
+id: developer-gorgone-rebound-mode
+title : Configuring Gorgone in rebound mode
+---
+
+This procedure describes how to configure Gorgone between a distant poller and a central server, via a rebound server.
```

Comment:
I had some usecases that come to mind, but I think asking consulting team about real world usecases would be a nice addition to this section